executor/executor_test.go

```diff
@@ -8456,7 +8417,14 @@ func (s testSerialSuite) TestTemporaryTableNoNetwork(c *C) {
 	// Index lookup
 	tk.HasPlan("select id from tmp_t where a = 1", "IndexLookUp")
 	tk.MustQuery("select id from tmp_t where a = 1").Check(testkit.Rows("1"))
+	tk.MustExec("rollback")
 
+	// Pessimistic lock
+	tk.MustExec("begin pessimistic")
```

Comment:
It seems that a failed case will cause retry and makes test hanging for a long time